anybox/recipe/odoo/runtime/session.py

```diff
@@ -158,6 +158,8 @@ def open(self, db=None, with_demo=False):
             db = config['db_name']
         if not db:
             db = ''  # expected value expected by Odoo to start defaulting.
+        else:
+            config['dbfilter'] = '$%s^' % db
```

Comment:
I like this addition, something I had been struggling with on my local multi-db environment!